engine/src/gldrv/gl_light_pick.cpp

```diff
@@ -30,10 +30,6 @@ struct light_key
         intensity_key = inte;
     }
 };
-static bool operator<( light_key tmp1, light_key tmp2 )
-{
-    return tmp1.intensity_key < tmp2.intensity_key;
-}
```

Comment:
It looks like, while this `operator <` technically wasn't **used** in the Vega Strike code, it was nevertheless necessary for implementing the template `less`. We'll need to put it back. And maybe add a `#pragma warn...` to disable the warning on it.

Other than that, your fixes look really good.

Reply:
Thanks for the vote of confidence, as well as the hint on how to make gcc ignore a warning. The code has been modified as per your instructions.